tests/integration/fixtures/split_and_sample_data/splitter_test_cases_and_fixtures.py

```diff
@@ -6,6 +6,7 @@
 
 import pandas as pd
 
+from great_expectations.core.metric_domain_types import MetricDomainTypes
```

Comment:
Why is this import being added without any other changes in the module?

Reply:
@Kilo59 Deleted -- must be a typo (as it is not used).  Good catch, thanks.